plugins/polio/js/src/domains/VaccineModule/StockManagement/StockVariation/Table/columns.tsx

```diff
@@ -35,6 +35,12 @@ export const useFormATableColumns = (
                 id: 'campaign',
                 sortable: true,
             },
+            {
+                Header: formatMessage(MESSAGES.round),
+                accessor: 'round_number',
+                id: 'round__number',
+                sortable: true,
+            },
```

Comment:
Since the backend may send `None`,  the front-end should display the `textPlaceholder` if the cell value is `null`